### DISCUSSION TITLE
Error message in New GOodf version 0.11

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#86] [General]
I am getting an error upon loading my latest sample-set in GO 
![image](https://github.com/GrandOrgue/GoOdf/assets/6999990/9c9ab960-e33d-4e23-93c7-de6606f073f6)

The only thing I changed using the new GOodf v0.11 is change the gain on this first stop and saving.

I loaded in the v10 and saved and the organ loaded as usual.... so it must be something in the GOodf v0.11.

**Comment by larspalo** [comment]
@Bumblebee001 I see that people are not reading my entries in the README, and CHANGELOG! ;-)

From the README:

`Please be aware that currently, due to changes in the GrandOrgue software, from the version 0.11.0 of GoOdf the .organ files written will require a GrandOrgue version 3.14+ to work! If compatibility with older versions of GO is important one must also use an older version of GoOdf (like 0.10.0) to write the odf.`

and in the CHANGELOG:

`Percussive and HasIndependentRelease values (default false) are inherited from [Organ]->[WindchestGroup]->[Rank]->[Pipe]. Note that this change in behaviour will cause odfs produced/saved with this version of GoOdf to be incompatible with earlier versions of GrandOrgue than 3.14+.`

So, in short it's due to changes in GrandOrgue behaviour and some features that didn't exist earlier. I did at one point consider adding an option to preserve writing the stop/rank percussive value to have more backwards compatibility, but in the end I considered that it was more important to work on other things...

**Comment by Bumblebee001** [comment]
And we are still no closer to getting the brightness property so much
needed. I have just downloaded the new grandorgue version and have not
explored it yet but I am not hopeful the voicing features are anywhere
closer to being more user friendly and practical judging by what I read and
understood on GO’s release notes. I stand to be corrected as I don’t always
appreciate what’s being described. Same unfortunately with the notes
related to GOodf referring to features I didn’t know even existed let alone
needing changes to be made to them and what they imply. I still consider
myself on the learning curve of samplesets creation even though I’ve been
doing this for over 10 years not without struggle and much help.
